### DISCUSSION TITLE
1.0 stable release

### DISCUSSION THREAD

**Discussion by ychescale9** [ReactiveCircus/FlowBinding#119] [General]
Thanks everyone for using FlowBinding, reporting bugs, providing feedback and sending PRs!

It's been over a year since the [initial release of FlowBinding](https://dev.to/ychescale9/binding-android-ui-with-kotlin-flow-22ok).

Over the last year we've added 4 new artifacts for AndroidX libraries, dropped our minimum SDK all the way back to API 14, and introduced a new [`InitialValueFlow` API for widgets that emit state changes](https://github.com/ReactiveCircus/FlowBinding/pull/98) (many thanks to @BenTilbrook and @svenjacobs). I feel like now it's right time for a 1.0 stable release 😃 . 

My current plan is to do the release sometime next week (around 🎄 ). Are there any bugs, features, API changes etc you'd like to see addressed before 1.0 release?

**Comment by svenjacobs** [comment]
Great news! Thanks for your work. I'm glad I could be of help 😃

**Comment by ychescale9** [comment]
Just released 1.0. Here's a short [blogpost](https://dev.to/ychescale9/flowbinding-1-0-44h).
